pkg/ring/lifecycler_test.go

```diff
@@ -246,17 +249,99 @@ func TestCheckReady(t *testing.T) {
 	flagext.DefaultValues(&ringConfig)
 	ringConfig.KVStore.Mock = &MockClient{}
 
+	tokens := Tokens([]uint32{1})
```

Comment:
Why do you need to define this variable, instead of simply doing the following below?

```
l1.setTokens(Tokens([]uint32{1}))
```